examples/auction/src/lib.rs

```diff
@@ -1,208 +1,201 @@
+//! # Implementation of an auction smart contract
+//!
+//! Account addresses can call the bid function to participate in the auction.
```

Comment:
I think the terminology we use is a bit different. We don't say that addresses call functions. We generally use "invoke" and it is not the address that does this, but an account. An address is a pointer to an account.

Reply:
I updated the sentence and I support the use of `invoke` for storage mutative functions in general.

Suggestion:
I have seen other projects (e.g. Hyperledger Fabric) using `invoke` for txs that execute a storage mutative function on chain while using `query` for non-storage mutative functions that are only `simulated` locally (e.g. the `view` function in this auction example). Concordium uses the word `invoke` also in the context of `simulating` a function execution locally:
`Invoking an instance is not a transaction and it does not change the state of a contract.`
We could improve on the use of `invoke` to avoid confusion.